wordpress-mobile/WordPress-Android
# Remove unused encrypted logging files

This PR mostly reverts the changes in https://github.com/wordpress-mobile/WordPress-Android/pull/11562 since we moved these to FluxC in https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1654.

**To test:**

* Crash the app using instructions from #12707
* Re-open the app
* Find the crash event in Sentry and grab its log uuid
* Use the uuid to verify the logs are successfully uploaded to MC

## Regression Notes
**1. Potential unintended areas of impact**

Accidentally removing encrypted logging from WPAndroid


**2. What I did to test those areas of impact (or what existing automated tests I relied on)**

I verified that encrypted logs are still working as expected using the test instructions above.


**3. What automated tests I added (or what prevented me from doing so)**

N/A

**PR submission checklist:**
